README.md

```diff
@@ -1 +1,30 @@
-# school-library
\ No newline at end of file
+# school-library
+> this project have the record of books and does the folowing
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+## Author
+
+👤 **Hadi Jafari**
+
+- GitHub: [@githubhandle](https://github.com/Hadi111jafari)
+- Twitter: [@twitterhandle](https://twitter.com/HadiJaf62814146)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdul-hadi-jafari)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
```

Comment:
- [ ] Kindly include the `Setup` section of your README.md file. This section should have commands for cloning this repository and running a file through the terminal (`ruby filename.rb`). 👍